### PR TITLE
OSD Tx Uplink Power support

### PR DESCRIPTION
**Pull request by dkustec**
This PR introduces the new OSD element 'Tx Uplink Power'.
The main PR for this is https://github.com/betaflight/betaflight/pull/10582
C/P from FW PR:

> The New OSD element 'Uplink Tx Power' is added in this PR.
> Currently, only CRSF is supported since the lack of internal implementation of other protocols, but using this method, they can be implemented easily later.
> Possible values are 25, 100, 250, 500 displayed as 'mW' and 1, 2 displayed as 'W'.
> Displaying this value can improve flight security when Dynamic Power is enabled. A user can check at any moment how many power levels are available when LQ is dropping.
> There is no character space in the MCM table for the "mW" or "LQpwr" symbol, so "MW" (or "W") is appended after the power state level.

### CODE REVIEW

File: locales/en/messages.json
Comment (etracer65) [inline locales/en/messages.json:5010]: Would suggest adding "for supporting radios" to avoid the expectation that this will work for all transmitter types.

File: src/js/tabs/osd.js
Comment (etracer65) [inline src/js/tabs/osd.js:455]: This doesn't match the behavior of the firmware change. First we should not unnecessarily constrain this to CRSF in the code. That would mean we'd need yet another Configurator change if this capability is added to another radio protocol in the firmware. Also the firmware does not display "---". It would display "0MW" for a protocol that doesn't currently support the uplink power.

File: locales/en/messages.json
Comment (dkustec) [reply locales/en/messages.json:5010]: Done.

File: src/js/tabs/osd.js
Comment (dkustec) [reply src/js/tabs/osd.js:455]: Thanks. Done.

**Comment by etracer65** [comment]
Looks good to me, but I'll let others review the technical implementation.

**Review by haslinghuis — APPROVED** [review]

**Review by mikeller — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> PASS
 - PR is not in draft -> PASS
 - Assigned to a milestone -> PASS
 - cooling off period lapsed (2/04/2021 19:02:41) -> PASS
 - commit count less or equal to three (1 commits) -> PASS
 - 'Don't merge' label NOT found -> PASS
 - 'Needs coordination with betaflight' label NOT found -> FAIL
 - At least one 'RN:' label found -> PASS
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (4 approvers) -> PASS

**Pull request merged**
